maas/plugins/maas_common.py

```diff
@@ -531,7 +531,7 @@ def status_ok(message=None, force_print=False):
 
 def metric(name, metric_type, value, unit=None):
     global METRICS
-    if len(METRICS) > 29:
+    if len(METRICS) > 49:
```

Comment:
I assume there is some historical math going on here? As in, the limit, minus the `status` line first?